### ISSUE TITLE
New Benchmarks

### ISSUE THREAD

**Issue by guarin** [lightly-ai/lightly#1197]
We are in the process of updating our benchmark code and adding benchmarks on the ImageNet dataset. The goal of this issue is to coordinate and track progress of the update.

The new benchmark code lives in [lightly/benchmarks](https://github.com/lightly-ai/lightly/tree/master/benchmarks/imagenet/resnet50) and initial focus is on reference implementations for the various SSL methods following the settings from the respective papers. We'll limit training to 100 epochs and small batch sizes (2x128) for now due to limited compute resources.

> **Note**: The benchmarking code is very much work in progress, there will be frequent and breaking changes.

## Implemented Methods

Methods are roughly ordered by importance (most important first).

- [x] SimCLR #1162 
- [ ] SwaV
- [ ] MAE
- [ ] DINO
- [ ] BYOL
- [ ] MoCov2
- [ ] BarlowTwins
- [ ] MSN
- [ ] PMSN
- [ ] DCL & DCLW
- [ ] SimMIM
- [ ] TiCo
- [ ] VICReg
- [ ] VICRegL
- [ ] NNCLR
- [ ] SMoG

## Contribute

Contributions are very welcome! You can help us with the following:
* Add one of the existing models to the new benchmarks. For this you can follow the SimCLR example.
* Let us know if you spot any mistake in the models or benchmarking code.
* Run benchmarks for different epochs / batch size.

Let us know if you are working on a model and we can help you out with reviews/compute.

**Comment by ibro45** [comment]
Will you be creating a tensorboard with training losses of all these networks? May be useful when a user wants to get an idea of how the loss behaves during training.

**Comment by guarin** [comment]
Hi, yes this is the goal :)
